Makefile

```diff
@@ -207,16 +207,12 @@ release: release-android release-ios ##@build Build release for Android and iOS
 
 build-fdroid: export BUILD_ENV = prod
 build-fdroid: export BUILD_TYPE = release
-build-fdroid: export ANDROID_ABI_SPLIT = false
-build-fdroid: export ANDROID_ABI_INCLUDE = armeabi-v7a;arm64-v8a;x86;x86_64
```

Comment:
Why are we removing this?

Reply:
to keep it consistent with how we build `APK` for playstore.
The values of `ANDROID_ABI_SPLIT` and `ANDROID_ABI_INCLUDE` would then be referred from `android/gradle.properties`

Reply:
I have however not tested this `fdroid` build change. I'll check and report back.

Reply:
But this is NOT for playstore, it's for F-Droid dude.

Reply:
alright I reverted this change.